hash/spectrum_cass.xml

```diff
@@ -106717,45 +106728,35 @@
 	</software>
 
 	<software name="plusdsys">
-		<description>Plus D System Tape (MGT)</description>
-		<year>19??</year>
-		<publisher>&lt;unknown&gt;</publisher>
+		<description>Plus D System Tape v2a</description>
+		<year>1987</year>
+		<publisher>Miles Gordon Technology</publisher>
 		<part name="cass" interface="spectrum_cass">
-			<dataarea name="cass" size="24362">
-				<rom name="plus d system tape (mgt).tzx" size="24362" crc="948c5cc9" sha1="1ef58a4d60c234fd187329ee1219ce969956f747"/>
+			<dataarea name="cass" size="24352">
+				<rom name="plus d system tape v2a.tzx" size="24352" crc="892644d5" sha1="6beb6507c01426e17679720604f4da4868b9ebfc"/>
 			</dataarea>
 		</part>
 	</software>
 
 	<software name="plusdsys_a" cloneof="plusdsys">
-		<description>Plus D System Tape v1.0</description>
-		<year>19??</year>
-		<publisher>&lt;unknown&gt;</publisher>
+		<description>Plus D System Tape v1</description>
+		<year>1987</year>
+		<publisher>Miles Gordon Technology</publisher>
 		<part name="cass" interface="spectrum_cass">
-			<dataarea name="cass" size="48402">
-				<rom name="plus d system tape v1.0.tzx" size="48402" crc="a69bd63c" sha1="dd353f31dc5567a12755889163ca2d9d6b7447a7"/>
+			<dataarea name="cass" size="24266">
+				<rom name="plus d system tape v1.tzx" size="24266" crc="198fb732" sha1="128ed4ad929a8e856d57092bf1d21484090484f6"/>
 			</dataarea>
 		</part>
 	</software>
 
+	<!-- data is just 3 bytes different from v2a, title screen shows v2a, might just be bad dump? -->
 	<software name="plusdsys_b" cloneof="plusdsys">
-		<description>Plus D System Tape v2.3</description>
-		<year>19??</year>
-		<publisher>&lt;unknown&gt;</publisher>
-		<part name="cass" interface="spectrum_cass">
-			<dataarea name="cass" size="24373">
-				<rom name="plus d system tape v2.3.tzx" size="24373" crc="6e3f966b" sha1="85bacfdd3b3d4d969c423857899d7d31d4b1db12"/>
-			</dataarea>
-		</part>
-	</software>
-
-	<software name="plusdsys_c" cloneof="plusdsys">
-		<description>Plus D System Tape v2a</description>
-		<year>19??</year>
-		<publisher>&lt;unknown&gt;</publisher>
+		<description>Plus D System Tape v2b?</description>
+		<year>1987</year>
+		<publisher>Miles Gordon Technology</publisher>
 		<part name="cass" interface="spectrum_cass">
-			<dataarea name="cass" size="24269">
-				<rom name="plus d system tape v2a.tzx" size="24269" crc="7e1a097c" sha1="1f702857d90dd49014f13aa86187152caf0d5a8b"/>
```

Comment:
What happened to this dump (Plus D System Tape v2a)?  It no longer appears in any software list.  Was it found to be bad or hacked?

Reply:
It was a duplicate (when extracted from the tzx the data was the same). After extensive searching the only genuine versions seem to be v1 and v2a. There is no v23 as some sources indicate. (MGT version numbers were always single digit, single letter.) Although one would assume there's a v2 (-non a) out there yet to be found. The v2b? is just 3 bytes different to v2a (and states v2a on the title screen), I think it's most likely just a bad dump of v2a so put a note to that effect in the soft list.
I actually meant to explain all that in the original PR, apologies.